gallery/how-to/managing-profile/managing-account.md

```diff
@@ -6,82 +6,94 @@ title:  PowerShell Gallery Account Settings
+   ![Login Account settings](../../Images/PSGallery_LoginAccountChange-2.png)
 
-3. You will see another prompt that you are signed in with the current account. Select "Sign out and sign in with a different account." 
+3. When you click **Next**, you see a message that you are signed in using the current account.
+   Click **Sign out and sign in with a different account**.
 
-![Login Account settings](../../Images/PSGallery_LoginAccountChange-3.png)
+   ![Login Account settings](../../Images/PSGallery_LoginAccountChange-3.png)
 
-4. You will then be prompted for the password of the new account, to complete the process. Once you enter the password, you will be returned to the Account Settings page, which will show that the login account has been updated.
+4. Enter the password of the new account. After entering the password, you are returned to the
+   Account Settings page showing you that the login account has been updated.
 
 
 ## Enable Two-Factor Authentication (2FA)
 
-Enabling two-factor authentication is a recommended security measure for all users who publish manually to the PowerShell Gallery. 
-This change will require that your login account has at least two forms of authentication registered for it. One will be the password, the other may be 
+Two-factor authentication is recommended for all users who publish manually to the PowerShell
+Gallery. To enable two-factor authentication, login account must have at least two forms of
+authentication registered. One is a password and the other forms can be:
 
-* A phone number than can receive texts;
-* Registration with an authenticator application on your phone, such as Microsoft Authenticator
+- A phone number that can receive text messages
+- A registered authenticator application, such as Microsoft Authenticator for your mobile phone
 
-Configuring what forms of authentication are used must be done via your AAD account information, or your Microsoft ID Account Security settings. 
+These forms of authentication must be configured in your AAD account information or in your
+Microsoft ID Account Security settings.
 
-Once you select "Enable", 2FA will be turned on for your account. 
-If you leave the PowerShell Gallery, you will be required to authenticate your login account with the two forms of authentication you have configured. 
+Once 2FA is enabled, you are required to authenticate using the configured forms of authentication
+every time you sign in to the PowerShell Gallery.
 
 > [!IMPORTANT]
-> Enabling two-factor authentication for the PowerShell Gallery will apply only to this site. However, if you are unable to log in, you may lose access to your PowerShell Gallery account. For that reason, it is highly recommended that you have at least three methods of authentication (password plus two others) associated with the login account.
-
+> Enabling two-factor authentication only applies to the PowerShell Gallery site.
```

Comment:
Wanted to be clear that enabling 2FA via this page would only apply to the PSGallery site. Suggesting:
Enabling two-factor authentication using ?this page? or account settings?...